onebusaway-android/build.gradle

```diff
@@ -27,11 +27,14 @@ apply plugin: 'com.android.application'
 
 repositories {
     jcenter()
-    mavenLocal()
     maven {
         // OBA Releases - for comparator to sort alphanumeric routes
         url "http://nexus.onebusaway.org/nexus/content/repositories/releases"
```

Comment:
Could we add the Amazon libraries to this Nexus repo so we could avoid them in the git repo all together?